stubgen_wrapper.py

```diff
@@ -108,13 +110,29 @@ def wrapper():
         idx = args.index("-o")
         args[idx + 1] = str(bindir / args[idx + 1])
 
+    if "-O" in args:
+        # we have an output directory, use its path instead relative to $(BINDIR),
+        # but in absolute form.
```

Comment:
Could you explain this comment? Since the output dir is relative to BINDIR, it cannot be absolute.

Reply:
I was copying a similar comment that can be found on line 108, but have now removed it